assets/cubyz/biomes/cave/stalagmite_cave.json

```diff
@@ -0,0 +1,13 @@
+{
+	"isCave" : true,
+	"maxHeight" : -512,
+
+	"chance" : 3,
```

Comment:
Please don't make these so common. I understand that it is easier to test if these biomes are more common, but this basically makes the regular caves obsolete.